README.md

```diff
@@ -28,29 +28,62 @@ application, each tutorial is constructed as hybrid theory and programming in an
 interactive environment, removing the gap between theory and implementation.
 
 If you have comments, questions, or would like to contribute to the project
-please feel free to email [me](mailto:dgasmith@gatech.edu).
+please see our [contributor guidelines]().
 
 ### Getting Started
 
-New users can either follow the
-[Getting Started](https://github.com/psi4/psi4numpy/blob/master/Tutorials/01_Psi4NumPy-Basics/1a_Getting-Started.ipynb)
-notebook or follow the less verbose directions below.
+1. Obtain required software:
+- [Psi4](http://psicode.org/psi4manual/1.1/build_obtaining.html)
```

Comment:
Yes, that's a good point.  I don't think there's any reason for them *not* to clone & compile Psi4, although that's certainly not the path of least resistance.